RosNet/RosReader/OpCode.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
This should not be needed as you are using `implicit global using` from C# 10 afaik

Reply:
We haven't gone though the using statements yet. A lot of them can generally be removed as they are unused. I will go through and purge some of them tomorrow